src/platform/plugins/private/presentation_panel/public/panel_component/presentation_panel_internal.tsx

```diff
@@ -170,3 +173,31 @@ export const PresentationPanelInternal = <
     </PresentationPanelHoverActions>
   );
 };
+
+/**
+ * if there is no reliance on EUI theme, then it is more performant to store styles as minimizable objects
+ * outside of the React component so that it is not parsed on every render
+ */
```

Comment:
See https://engineering.deptagency.com/optimizing-emotion-in-react#optimization-i-avoid-doing-construction-of-styles-in-render-functions for a more thorough explanation here